cmake/release/linux/rpm/iroha.cmake

```diff
@@ -0,0 +1,14 @@
+find_program(RPMBUILD rpmbuild)
+if(RPMBUILD STREQUAL "RPMBUILD-NOTFOUND")
+  message(WARNING "linux-rpm can not be build without `rpmbuild`. ")
+else()
+  # add deb to list of generators
+  list(APPEND CPACK_GENERATOR RPM)
+
+  message(STATUS "[package] linux-rpm standalone will be packaged.")
+
+  SET(CPACK_RPM_PACKAGE_LICENSE "${CMAKE_SOURCE_DIR}/LICENSE")
+  SET(CPACK_RPM_PACKAGE_REQUIRES "libstdc++5, libpq-dev, libtbb-dev, libboost-system-dev, libboost-filesystem-dev, libc-ares-dev")
+endif()
+
+#   SET(CPACK_RPM_PACKAGE_REQUIRES "libstdc++5, libpq-dev (>= 9.5), libtbb-dev (>= 4.4), libboost-system-dev (>= 1.56.0), libboost-filesystem-dev (>= 1.56.0), libc-ares-dev (>= 1.10.0)")
```

Comment:
what is that

Reply:
It is there to not forget required versions. 
Actually, can be safely removed.